### PR TITLE
Add the note about cyborg objectives before transferring the mind so …

### PR DESCRIPTION
**Pull request by Ikarrus**
…it shows up when their memory is printed out to them.

Some players were thinking they still had objectives to follow because they didn't see the line.

### CODE REVIEW

File: code/game/objects/items/robot/robot_parts.dm [hunk 1]
```diff
@@ -222,11 +222,11 @@
 				if(ticker.mode.config_tag == "malfunction") //Don't let humans get a cyborg on their side during malf, for balance reasons.
 					O.set_zeroth_law("<span class='danger'>ERROR ER0RR $R0RRO$!R41.%%!!(%$^^__+ @#F0E4'STATION OVERRUN, ASSUME CONTROL TO CONTAIN OUTBREAK#*�&110010</span>")
 
-			BM.mind.transfer_to(O)
-
 			if(O.mind && O.mind.special_role)
```
Comment (RemieRichards) [inline code/game/objects/items/robot/robot_parts.dm]: Sorry to ask this but did you actually test this? The way the code reads it's asking if the Cyborg is an antag and then adding it to the Cyborg's mind... except in your changed edition the cyborg is NOT the same mind as the person being borged... it's most likely a null mind.

tl;dr I think you've misunderstood the ordering of this code/forgotten to update the previous ```O```s to ```BM```s.
